lua/better_escape.lua

```diff
@@ -12,11 +13,20 @@ local flag = false
 local previous_chars = {}
 local first_chars = {}
 local second_chars = {}
+local timer = nil
 
 local function start_timeout()
   flag = true
+
+  if timer then
+    uv.stop_timer(timer)
+    uv.close(timer)
+    timer = nil
+  end
+  
   vim.defer_fn(function()
     flag = false
+    timer = nil
```

Comment:
Isn't `timer` `nil`at the beginning and always stays `nil`?

Reply:
sorry,I forgot to set timer variable by `timer = vim.defer_fn()`. Fixed now.